mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -190,6 +192,146 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class MultiViewWrapper():
+    """Wrap transformation from single-view into multi-view.
+
+    Args:
+        transforms (list[dict]): A list of dict specifying the transformations
+            for the monocular situation.
+        collected_keys (list[str]): Collect information in transformation
+            like rotate angles, crop roi, and flip state.
+    """
+
+    def __init__(self, transforms, collected_keys=[]):
+        self.t = Compose(transforms)
+        self.collected_keys = collected_keys
+
+    def __call__(self, input_dict):
+        for key in self.collected_keys:
+            input_dict[key] = []
+        for img_id in range(len(input_dict['img'])):
+            process_dict = dict(img=input_dict['img'][img_id])
+            process_dict = self.t(process_dict)
+            input_dict['img'][img_id] = process_dict['img']
+            for key in self.collected_keys:
+                input_dict[key].append(process_dict[key])
+        return input_dict
+
+
+@PIPELINES.register_module()
+class RandomCrop(MMDetRandomCrop):
```

Comment:
Since it add constraint on the original RandomCrop, suggest using a different name, like RangeLimitedRandomCrop